pages/about/AboutContextHandler.inc.php

```diff
@@ -51,6 +51,55 @@ function index($args, $request) {
 
 		$this->setupTemplate($request);
 		$templateMgr = TemplateManager::getManager($request);
+
+		$templateMgr->assign('journal', $context);
+		$templateMgr->assign('description', $context->getLocalizedSetting('description'));
+		$templateMgr->assign('aboutJournal', $context->getLocalizedSetting('aboutJournal'));
```

Comment:
I prefer to do e.g. `{$currentContext->getLocalizedSetting('aboutJournal')|...} in the template -- less spaghetti divided between PHP and template -- but just my preference.

For long lists of template assignments, I've started using the following:

    $templateMgr->assign(array(
        'a' => $a,
        ...
    ));